book/en/src/migration_v1_v2.md

```diff
@@ -2,7 +2,7 @@
 
 Migrating a project from RTIC `v1.0.x` to `v2.0.0` involves the following steps:
 
-1. `v2.0.0` requires [`#![type_alias_impl_trait]`](https://github.com/rust-lang/rust/issues/63063) and Rust Nightly.
+1. `v2.0.0` works on Rust Stable from 1.75.
```

Comment:
I imagine you'll release this as 2.1.0 or 2.0.1 or something? So maybe it should make clear that 2.0.0 requires nightly but 2.1.0 only results Rust 1.75 or higher.

Reply:
Thank you for the feedback, it is fixed now!